docs/source/openvino/export.mdx

```diff
@@ -31,7 +31,7 @@ Check out the help for more options:
 
 ```text
 usage: optimum-cli export openvino [-h] -m MODEL [--task TASK] [--framework {pt}] [--trust-remote-code]
-                                   [--weight-format {fp32,fp16,int8,int4,mxfp4,nf4,cb4}]
+                                   [--weight-format {fp32,fp16,int8,int4,mxfp4,nf4,fp4,fp8_e4m3,cb4}]
```

Comment:
In my opinion we should rename `fp8_e4m3` to `f8e4m3` to keep being aligned with `f8e4m3` option for `--quant-mode`.

cc @ljaljushkin

Reply:
to be honest, don't like that optimum and nncf names are different and not aligned. 
not clear why quant-mode in optimum is f8e4m3, but nncf has `QuantizationMode.FP8_E4M3`

MXFP4, MXFP8 and NVFP4 are established names and don't follow `mxf4_e2m1` and `nvf4_e2m1` format convention.
fp32 and fp16 are also not f32 and f16. not sure it was a good choice for `--quant-mode`. Can we reconsider it in optimum? will it affect someone?

Reply:
@helena-intel could you please provide your feedback on this? The proposal is to rename some low precision data type names in optimum-intel to be consistent with their corresponding names in NNCF, for example `f8e4m3` -> `fp8_e4m3`.